client/me/security-checkup/account-recovery-phone.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getAccountRecoveryPhone,
+	isAccountRecoveryPhoneActionInProgress,
+	isAccountRecoveryPhoneValidated,
+} from 'state/account-recovery/settings/selectors';
+import { getOKIcon, getWarningIcon } from './icons.js';
+import QueryAccountRecoverySettings from 'components/data/query-account-recovery-settings';
+import SecurityCheckupNavigationItem from './navigation-item';
+
+class SecurityCheckupAccountRecoveryPhone extends React.Component {
+	static propTypes = {
+		accountRecoveryPhone: PropTypes.object,
+		accountRecoveryPhoneActionInProgress: PropTypes.bool,
+		accountRecoveryPhoneValidated: PropTypes.bool,
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const {
+			accountRecoveryPhone,
+			accountRecoveryPhoneActionInProgress,
+			accountRecoveryPhoneValidated,
+			translate,
+		} = this.props;
+
+		if ( accountRecoveryPhoneActionInProgress ) {
+			return (
+				<React.Fragment>
+					<QueryAccountRecoverySettings />
+					<SecurityCheckupNavigationItem isPlaceholder={ true } />;
+				</React.Fragment>
+			);
+		}
+
+		let icon, description;
+
+		if ( ! accountRecoveryPhone ) {
+			icon = getWarningIcon();
+			description = translate( 'You do not have a recovery SMS number.' );
+		} else if ( ! accountRecoveryPhoneValidated ) {
+			icon = getWarningIcon();
+			description = translate(
+				'You still need to verify your recovery SMS number: {{strong}}%(recoveryPhoneNumber)s{{/strong}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'You still need to validate your recovery SMS number: {{strong}}%(recoveryPhoneNumber)s{{/strong}}' )` *ES Score: 8*